adopterAPI.json

```diff
@@ -342,19 +349,18 @@
             "format": "uuid"
           },
           "userName": {
-            "type": "string",
-            "nullable": true
+            "type": "string"
           },
           "phoneNumber": {
             "type": "string",
             "nullable": true
           },
           "email": {
-            "type": "string",
-            "nullable": true
+            "type": "string"
           },
           "address": {
-            "$ref": "#/components/schemas/NewAddress"
+            "$ref": "#/components/schemas/NewAddress",
+            "nullable": true
           },
```

Comment:
Do we want `phoneNumber` and `address` to be nullable? It doesn't affect anything, so we really just need to choose what we prefer.